Makefile

```diff
@@ -49,8 +49,8 @@ LOGGLY_TOKEN_MOBILE := d730c074-1f0a-415d-8d71-1ebf1d8bd736
 
 FIRETWEET_MAIN_DIR ?= ../firetweet/firetweet/src/main/
 
-CHAINED_YAML := chained.yaml
-CHAINED_PATH := installer-resources/chained.yaml
+LANTERN_YAML := lantern.yaml
```

Comment:
Embed lantern.yaml in installer, and copy it to config directory if there's no existing good config.